style.css

```diff
@@ -458,3 +457,151 @@
 .icon {
   margin-right: 20px;
 }
+
+.contact {
+  width: 100%;
+  background: #6070ff;
+  background-image: url(.Contact-me.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: 143px 11px;
+  border-top-left-radius: 80px;
+  margin-top: -70px;
+  padding: 103px 12px;
+  display: flex;
+  flex-direction: column;
+  gap: 100px;
+  align-items: center;
+}
+
+.contact-1 {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.contact #contact-me {
+  color: #fff;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+}
+
+.contact #contact-messeage {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  font-family: "Poppins", sans-serif;
+}
+
+.contact ul {
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  align-items: center;
+}
+
+.contact .my-contact-form {
+  margin-top: 20px;
+  order: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 14px;
+  padding-left: 0;
+}
+
+#name,
+#email,
+#messages {
+  outline: none;
+}
+
+input,
+textarea {
+  border-top-style: hidden;
+  border-right-style: hidden;
+  border-left-style: hidden;
+  border-bottom-style: hidden;
+  left: 16px;
+  right: 38px;
+  top: 15px;
+  bottom: 15px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.contact .input-data {
+  width: 327px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 15px 38px 15px 16px;
+}
+
+.contact #user-name,
+#email-addres {
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+}
+
+.contact input::placeholder {
+  color: #172b4d;
+}
+
+.contact textarea::placeholder {
+  color: #b3bac5;
+}
+
+.contact textarea {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  resize: none;
+}
+
+.first-button {
+  width: 108px;
+  height: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.contact .first-button {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 12px 16px;
+  width: 140px;
+  height: 48px;
+  left: 24px;
+  top: 5317px;
+}
```

Comment:
- [ ] The button shouldn't have a fixed width let it contain the text to match the design, kindly change that: 😁

![image](https://user-images.githubusercontent.com/20567503/177543123-8f989c7f-e504-4b5f-a71d-9c029ec37a3c.png)

- [ ] Please add styling for the other states of the button (`active`, `hover` and `disabled`) as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design: 😁

![image](https://user-images.githubusercontent.com/20567503/176999701-bcd4478c-ec15-4f89-a578-f0f1b179e3db.png)